packages/eslint-plugin-pf-codemods/src/rules/v6/componentGroupsMultiContentCardRemoveProps/component-groups-multi-content-card-remove-props.ts

```diff
@@ -0,0 +1,12 @@
+import { renameProps } from "../../helpers";
+
+// https://github.com/patternfly/react-component-groups/pull/145
+
+const renames = {
+  MultiContentCard: { leftBorderVariant: "", withHeaderBorder: "" },
+};
+
+module.exports = {
+  meta: { fixable: "code" },
+  create: renameProps(renames, "@patternfly/react-component-groups"),
```

Comment:
I think using the `renameProps` helper won't work with default imports. These components in react-component-groups have also default exports so they can be imported like this:

`import MyCard from '@patternfly/react-component-groups/dist/dynamic/MultiContentCard';`

I ran to this issue yesterday and I first wanted to rewrite the `getFromPackage` helper, which is used internally by `renameProps` to also handle ImportDefaultSpecifiers, but it would be too painful, so I decided to do a custom codemod instead (https://github.com/patternfly/pf-codemods/pull/711/files#diff-24111f58cfc0f6b2a81d0fa8599c5b82889b4f88666e8c4e8897694235a6ca59)